Themes/default/scripts/alerts.js

```diff
@@ -51,7 +51,7 @@ var updateAlerts = function ()
 		$.each(alerts, function(index, item)
 		{
 			var notification = notify.createNotification(new_alert_title, {
-				body: $(item).find('div.details:first > span').text(),
+				body: $(item).find('div.details:first span.alert_text').text(),
 				icon: smf_images_url + '/blank.png'
```

Comment:
Also, shouldn't we allow admin to choose an icon for the desktop notifications instead of always a blank image ?

Reply:
I'd say just use the OpenGraph image, falling back to the favicon.